### DISCUSSION TITLE
Why is there a significant difference in the maximum number of epochs required for different segmentation tasks?

### DISCUSSION THREAD

**Discussion by AustinYuAo** [Project-MONAI/MONAI#7292] [General]
Hello, sorry to bother you. When comparing Swin-UNetR and nnU-Net, I noticed that Swin-UNetR requires a significantly different maximum number of epochs for different segmentation tasks, while nnU-Net uses the same maximum epoch value. Can you help me resolve this confusion?

**Comment by KumoLiu** [comment]
Hi @AustinYuAo, in theory, we want the task to converge at the end, and if the epoch num is big enough, it will certainly converge, and the values after convergence should not be significantly different.
